README.md

```diff
@@ -0,0 +1,197 @@
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 EasyClinic API app <a name="about-project"></a>
+
+**EasyClinic** is a back-end application, for our Final Capstone Project. It is based on an app to book an appointment with a doctor from a wide health network.
+
+**EasyClinic** front-end application is available [here](https://github.com/oovillagran/final-capstone-front-end).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+- PostgreSQL
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Use of Ruby on Rails framework.**
+- **Back-end application**
+- **Use of Kanban board**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo & preview <a name="live demo"></a>
+
+### API Documentation(#api-documentation-)
+Here you can see the API Documentation, in [render](https://rails-xdb0.onrender.com/api-docs/index.html).
+
+### API URL 
+You can use this URL address to consump data from our API using [localy](localhost:3000/) or using the URL provided by render [https://rails-xdb0.onrender.com](https://rails-xdb0.onrender.com)
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Follow these steps in order to run ruby code.
+
+### Prerequisites
+
+First, you need to ensure that you have Rails and Ruby installed on your computer.
+
+### Setup
+
+It varies depending on your operating system so here are some links to help you install it on [Windows](https://gorails.com/setup/windows/10) and on [Ubuntu](https://www.ruby-lang.org/en/documentation/installation/#apt).
+
+For MacOS run the following commands
+
+```sh
+  brew install rbenv ruby-build
+  # Add rbenv to bash so that it loads every time you open a terminal
+  echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+  source ~/.bash_profile
+
+  # Install Ruby
+  rbenv install 3.0.1
+  rbenv global 3.0.1
+  ruby -v
+```
+
+This example shows how to install Ruby 3.0.1 which was the latest version in April 2021, but you can check to see if there is a newer version [here](https://www.ruby-lang.org/en/downloads/releases/). Iy is also assumed that you have [homebrew](https://brew.sh/) already installed on your Mac.
+
+### Usage
+
+Once you have ruby installed, [clone this repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+To run the project, execute the following command in the terminal:
+```
+bundle install
+```
+```
+rails server
+```
+To see all API Endpoints available, in your preferred browser, navigate to:
+```
+https://rails-xdb0.onrender.com/api-docs/index.html
+```
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍